app/views/layouts/_header.html.haml

```diff
@@ -0,0 +1,3 @@
+.pull-right
+  = link_to 'Login', '#', class: 'btn btn-primary'
```

Comment:
shouldn't be doing this here, this will be handled when the user authentication is added in